content/docs/command-reference/exp/run.md

```diff
@@ -134,6 +141,17 @@ CPU cores).
 - `-j <number>`, `--jobs <number>` - run this `number` of queued experiments in
   parallel. Only applicable when used in conjunction with `--run-all`.
 
+- `-r <commit>`, `--rev <commit>` - continue an experiment from a specific
+  checkpoint name or hash (`commit`). Required for explicitly resuming
+  checkpoints in queued or temp dir runs.
+
+- `--reset` - restart a checkpoint experiment from scratch (resets existing
+  checkpoints to the initial state).
+
+- `-f`, `--force` - reproduce pipelines even if no changes were found (same as
+  `dvc repro -f`). For `checkpoint` experiments, this restarts form scratch,
+  like `--reset`.
```

Comment:
Can you please just confirm this is correct? Ty

Reply:
This is also outdated, `--reset` must be explicitly used to restart checkpoints from scratch. Also it should be noted that using `--reset` now implicitly sets `-f/--force` so that the checkpoints which were previously generated will be explicitly reproduced again (instead of loaded from run cache).